### PR TITLE
Adding a file about cannon beach

### LINKED ISSUE #63: Create a new file for pbennyb hometown
_**Add file for Cannon Beach, Oregon**_
- [ ] Create a branch
- [ ] Add file
- [ ] Create a commit
- [ ] Open pull request
- [ ] Have a discussion
- [ ] Merge pull request

### PR DESCRIPTION
**Pull request by pbennyb**
Pull request closes #63

### CODE REVIEW

File: places/cannon-beach.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#Cannon Beach, Oregon
+###Best Beaches in the Area
```
Comment (pbennyb) [inline places/cannon-beach.md:2]: Test comment

Comment (bhooshan786) [inline places/cannon-beach.md:2]: Hi

**Comment by lsigle** [comment]
Looks good :ship:

**Comment by bhooshan786** [comment]
looks good :shipit: @pbennyb

**Pull request merged**